src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -587,11 +596,14 @@ object SbtBundle extends AutoPlugin {
   private def formatServiceName(serviceName: String): String =
     s"""        service-name  = "$serviceName""""
 
-  private def formatEndpoints(endpoints: Map[String, Endpoint]): String = {
+  private def formatEndpoints(bundleConfVersion: BundleConfVersions.Value, endpoints: Map[String, Endpoint]): String = {
     val formatted =
       for {
         (label, Endpoint(bindProtocol, bindPort, services, serviceName, acls)) <- endpoints
       } yield {
+        if (acls.exists(_.nonEmpty) && bundleConfVersion != BundleConfVersions.V_1_2_0)
```

Comment:
It would be nice to check if the bundleConfVersion is 1.2.0 or above. This will help not introducing errors when we release 1.3.0.